src/style.css

```
@@ -1 +1,74 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  /* border: 3px solid orangered */
+}
+
+.nav__ul {
+  display: flex;
+  align-items: center;
+  /* border: 3px solid blueviolet; */
+  list-style: none;
+}
+
+.nav__li {
+  margin: 0 10px;
+}
+
+.nav__li:nth-child(8) {
+  margin-right: 20px;
+}
+
+.nav__a {
+  color: black;
+  font-size: 0.8rem;
+  text-decoration: none;
+  text-transform: uppercase;
+  height: 60px;
+  padding: 20px 0;
+ 
+}
+
+.nav .is-active {
+  color: #00acdc;
+}
+
+.nav__a:hover {
+  color: #00acdc;
+}
+
+.nav__li:first-child {
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-color: #00acdc;
+  padding: 2px;
+  width: 40px;
```

don't hardcode the width
```suggestion
  width: 100%;
```